template/go/Dockerfile

```diff
@@ -1,63 +1,63 @@
-FROM --platform=${TARGETPLATFORM:-linux/amd64} ghcr.io/openfaas/classic-watchdog:0.1.5 as watchdog
-FROM --platform=${BUILDPLATFORM:-linux/amd64} golang:1.15-alpine3.13 as builder
-
-ARG TARGETPLATFORM
-ARG BUILDPLATFORM
-ARG TARGETOS
-ARG TARGETARCH
-
-# Required to enable Go modules
-RUN apk add --no-cache git
-
-# Allows you to add additional packages via build-arg
-ARG ADDITIONAL_PACKAGE
-ARG CGO_ENABLED=0
-ARG GO111MODULE="off"
-ARG GOPROXY=""
-ARG GOFLAGS=""
-
-COPY --from=watchdog /fwatchdog /usr/bin/fwatchdog
-RUN chmod +x /usr/bin/fwatchdog
-
-ENV CGO_ENABLED=0
-
-WORKDIR /go/src/handler
-COPY . .
-
-# Add user overrides to the root go.mod, which is the only place "replace" can be used
-RUN cat function/GO_REPLACE.txt >> ./go.mod || exit 0
-
-# Run a gofmt and exclude all vendored code.
-RUN test -z "$(gofmt -l $(find . -type f -name '*.go' -not -path "./vendor/*" -not -path "./function/vendor/*"))" || { echo "Run \"gofmt -s -w\" on your Golang code"; exit 1; }
-
-WORKDIR /go/src/handler/function
-
-RUN GOOS=${TARGETOS} GOARCH=${TARGETARCH} CGO_ENABLED=${CGO_ENABLED} go test ./... -cover
-
-WORKDIR /go/src/handler
-
-RUN GOOS=${TARGETOS} GOARCH=${TARGETARCH} CGO_ENABLED=${CGO_ENABLED} \
-    go build --ldflags "-s -w" -a -installsuffix cgo -o handler .
-
-FROM --platform=${TARGETPLATFORM:-linux/amd64} alpine:3.12
-RUN apk --no-cache add ca-certificates \
-    && addgroup -S app && adduser -S -g app app \
-    && mkdir -p /home/app \
-    && chown app /home/app
-
-WORKDIR /home/app
-
-COPY --from=builder /usr/bin/fwatchdog         .
-COPY --from=builder /go/src/handler/function/  .
-COPY --from=builder /go/src/handler/handler    .
-
-RUN chown -R app /home/app
-
-USER app
-
-ENV fprocess="./handler"
-EXPOSE 8080
-
-HEALTHCHECK --interval=3s CMD [ -e /tmp/.lock ] || exit 1
-
-CMD ["./fwatchdog"]
+FROM --platform=${TARGETPLATFORM:-linux/amd64} ghcr.io/openfaas/classic-watchdog:0.1.5 as watchdog

```

Comment:
Why have so many lines changed?

Reply:
I don't know, it is essentially all same, my assumption is CR LF and LF stuff on that file creates git to do this

Reply:
https://stackoverflow.com/questions/19593909/git-diff-sees-whole-file-as-changed-when-its-not

Reply:
Are you a Windows user primarily?